tools/release/README.md

```diff
@@ -52,6 +52,8 @@ A Quality Assurance Engineer from each product group needs to [confirm that thei
 
 **3. Merge changelog and version bump**
 
+Manally increment the [Helm chart]([url](https://github.com/fleetdm/fleet/blob/main/charts/fleet/Chart.yaml)) patch version by 1 and push to your changelog PRs.
```

Comment:
If the chart changes you update the minor, if only the app version changes you up the patch

Reply:
Ah right, thanks.